package.json

```diff
@@ -39,12 +39,11 @@
     "chai-as-promised": "^7.1.1",
     "ethereumjs-tx": "^1.3.3",
```

Comment:
I think it is a good idea to fix the version to only one version (I mean, upgrade the versions and remove `^`). What do you think?

Reply:
Mmmm maybe it is but we already have the `package-lock.json` file to be sure the versions we used are maintain when the `npm install` is executed so i think we don't really need that.